server/src/main.rs

```diff
@@ -67,7 +89,7 @@ fn main() {
         // The errors are consciously ignored as the program is terminating.
         let binary_path = std::env::current_exe().unwrap();
         let binary_name = binary_path.file_name().unwrap();
-        println!("usage: {} data_folder", binary_name.to_str().unwrap());
+        error!(" Usage: {} data_folder", binary_name.to_str().unwrap());
```

Comment:
Should the usage message really be a log message as it simply informs the user what arguments are required to start the system? For example for Apache Cassandra the usage message is:

 `Usage: ./cassandra [-f] [-h] [-p pidfile] [-H dumpfile] [-E errorfile]`

Despite using formatted logging through SLF4J at runtime. I propose we write usage to `eprintln!`.

Reply:
@CGodiksen Opinion?

Reply:
That sounds fine.

Reply:
Fixed

Reply:
Looks like you accidentally used `println!` instead of `eprintln!`.

Reply:
No, my latest push is eprintln!

Reply:
Sorry my mistake, somehow I looked at a version of the branch without that commit.